core/types/proto_block.proto

```diff
@@ -222,6 +223,14 @@ message ProtoKeys {
   repeated bytes keys = 1;
 }
 
+message ProtoKeyValue {
```

Comment:
i already left a comment in the accessors, not sure if we need two types since we can do maps in protobuf, but if there are other use cases i am wrong